### PR TITLE
BREAKING CHANGE: remove createElement usage in react-native-web

### LINKED ISSUE #355: v2.2.1 - Attempted import error: 'createElement' is not exported from react-native-web
Problem:

When running `expo build:web`, I receive the error below

Dependencies:
- picker v2.2.1
- react-native-web 0.14.10 
- expo 43

Solution:
- downgrade to 2.1.0

```
./node_modules/@react-native-picker/picker/dist/module/Picker.web.js:34:2
Attempted import error: 'createElement' is not exported from 'react-native-web/dist/index' (imported as 'ReactNativeWeb').
  32 |
  33 | const createElement =
> 34 |   ReactNativeWeb.createElement || ReactNativeWeb.unstable_createElement;
     |  ^
  35 |
  36 | const Select = forwardRef((props: any, forwardedRef) =>
  37 |   createElement('select', props),

./node_modules/@react-native-picker/picker/dist/module/PickerItem.js:22:2
Attempted import error: 'createElement' is not exported from 'react-native-web/dist/index' (imported as 'ReactNativeWeb').
  20 |
  21 | const createElement =
> 22 |   ReactNativeWeb.createElement || ReactNativeWeb.unstable_createElement;
     |  ^
  23 |
```

Thank you!

### PR DESCRIPTION
**Pull request by listiani13**
Remove createElement usage & use `unstable_createElement` in web instead [as it has been renamed in react-native-web v0.12.0](https://github.com/necolas/react-native-web/releases/tag/0.12.0)

Resolve #355

Tested by running example in web, as I'm only changing the web part
Kapture 2022-01-04 at 20 27 34

### CODE REVIEW

**Comment by marcod1419** [comment]
Thanks for making this! My team and I are looking forward to this fix being merged. 🎉

**Comment by Maker-Mark** [comment]
@marcod1419 Any idea when this will be released?

**Comment by marcod1419** [comment]
> @marcod1419 Any idea when this will be released?

Not sure, I'm not a maintainer on this project. Sorry for the confusion!

**Comment by Maker-Mark** [comment]
> > @marcod1419 Any idea when this will be released?
> 
> Not sure, I'm not a maintainer on this project. Sorry for the confusion!

Ahh, I read your comment in the wrong context 🤣 
I second what you said!  The team and I are looking forward to this fix being merged. 🎉

**Review by Naturalclar — APPROVED** [review]
Thanks! this will cause breaking change for those projects that are using older versions of react-native-web, so I'll make a major version release

**Comment by Maker-Mark** [comment]
@Naturalclar When are you going to make a release with the fix? I'd like to use the release version over this commit.

**Comment by mohammadgharouni** [comment]
I'm also waiting for this release...

**Comment by klifish** [comment]
Waiting for this release +1

**Comment by davidseek** [comment]
+1

**Pull request merged**
